core/src/main/java/io/aiven/klaw/service/OperationalRequestsService.java

```diff
@@ -203,14 +207,21 @@ private List<OperationalRequestsResponseModel> getOperationalRequestModels(
         rolesPermissionsControllerService.getApproverRoles("TOPICS", tenantId);
     List<UserInfo> userList = manageDatabase.getUsersPerTeamAndTenant(userTeamId, tenantId);
 
-    for (OperationalRequest topicReq : operationalRequestList) {
+    for (OperationalRequest operationalRequest : operationalRequestList) {
       operationalRequestModel = new OperationalRequestsResponseModel();
-      copyProperties(topicReq, operationalRequestModel);
-      operationalRequestModel.setRequestStatus(RequestStatus.of(topicReq.getRequestStatus()));
-      operationalRequestModel.setOperationalRequestType(topicReq.getOperationalRequestType());
+      copyProperties(operationalRequest, operationalRequestModel);
+      operationalRequestModel.setTeamId(operationalRequest.getRequestingTeamId());
+      operationalRequestModel.setRequestStatus(
+          RequestStatus.of(operationalRequest.getRequestStatus()));
+      operationalRequestModel.setOperationalRequestType(
+          operationalRequest.getOperationalRequestType());
+      if (operationalRequest.getResetTimeStamp() != null) {
+        operationalRequestModel.setResetTimeStampStr(
+            operationalRequest.getResetTimeStamp().toString() + " (UTC)");
```

Comment:
If we add (UTC) it will mean the FE can't change that into localtime and we will always display the UTC time what do you think?

Reply:
Originally we are taking the request with UTC, so thought we will display it back with UTC

Reply:
So if we return the time to the FE we can localise in coral/angular

Reply:
Yes I think it can be localised in FE. @aindriu-aiven